controllers/argocd/deployment.go

```diff
@@ -43,7 +45,9 @@ func getArgoCDRepoServerReplicas(cr *argoprojv1a1.ArgoCD) *int32 {
 	return nil
 }
 
-// getArgoCDServerReplicas will return the size value for the argocd-server replica count.
+// getArgoCDServerReplicas will return the size value for the argocd-server replica count if it 
+// has been set in argocd CR. Otherwise, nil is returned if the replicas is not set in the argocd CR or
```

Comment:
```suggestion
// getArgoCDServerReplicas will return the size value for the argocd-server replica count if it
// has been set in argocd CR. Otherwise, nil is returned if the replicas is not set in the argocd CR or
```